java/test/src/main/java/org/ray/api/test/BaseTest.java

```diff
@@ -1,27 +1,31 @@
 package org.ray.api.test;
 
+import java.lang.reflect.Method;
 import org.ray.api.Ray;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.testng.annotations.AfterMethod;
 import org.testng.annotations.BeforeMethod;
 
 public class BaseTest {
 
+  private static final Logger LOGGER = LoggerFactory.getLogger(BaseTest.class);
+
   @BeforeMethod
-  public void setUp() {
+  public void setUpBase(Method method) {
+    LOGGER.info("===== Running test: "
+        + method.getDeclaringClass().getName() + "." + method.getName());
```

Comment:
Also log the `current mode` in this log?
==== Running test: A.b [Cluster]

Reply:
probably not very useful. we can also find the mode from the command.